implementations/rust/ockam/ockam_command/src/forwarder/create.rs

```diff
@@ -31,7 +31,7 @@ impl CreateCommand {
         let port = match cfg.select_node(&cmd.node_name) {
             Some(cfg) => cfg.port,
             None => {
-                eprintln!("No such node available. Run `ockam node list` to list available nodes");
+                eprintln!("No such node available.  Run `ockam node list` to list available nodes");
```

Comment:
I've seen this double-space after a period all over the codebase. Is this some formatting option you use for a reason? Do we want to do this?

Reply:
@adrianbenavides I just wanted it to be consistent. @spacekookie thoughts?

Reply:
I am personally a fan of double spaces after periods.  But I also don't think we're currently consistent about it at all.  Either way works for me :sweat_smile: